frontend/src/features/admin-form/create/workflow/components/WorkflowContent/WorkflowContent.tsx

```diff
@@ -12,16 +14,31 @@ export const WorkflowContent = (): JSX.Element | null => {
   const { formWorkflow, isLoading } = useAdminFormWorkflow()
 
   if (isLoading) return null
-
   return (
-    <Stack color="secondary.500" spacing="2.75rem">
+    <Stack color="secondary.500" spacing="2.75rem" mt="1.5rem">
+      {/* <HeaderBlock /> */}
+      <Box
+        bg="white"
+        border="1px solid"
+        borderColor="neutral.300"
+        borderRadius="4px"
+        padding="1.5rem"
+      >
+        <Stack gap={'1.5rem'}>
+          <Text as="h2" textStyle="h2">
+            Workflow
+          </Text>
+          <Divider />
+          <StatusTrackerToggle />
+        </Stack>
+      </Box>
       <Stack spacing="0" divider={<WorkflowStepBlockDivider />}>
         {formWorkflow?.map((step, i) => (
           <WorkflowBlockFactory key={i} stepNumber={i} step={step} />
         ))}
         <NewStepBlock />
       </Stack>
-      <WorkflowCompletionMessageBlock />
+      {formWorkflow?.length == 0 ? null : <WorkflowCompletionMessageBlock />}
```

Comment:
```suggestion
      {formWorkflow?.length ? <WorkflowCompletionMessageBlock /> : null}
```
usually, in our code, null is the secondary clause

Reply:
Nice - code looks cleaner with this change